### ISSUE TITLE
[P1 HIGH] Fix CDP session memory leak + verify Chrome UA version consistency

### ISSUE THREAD

**Issue by MichaelEasycopeu** [nikotheclawd/vanta#28]
### 1. CDP session leak (playwrightEngine.js:1115-1140)
`applyCdpDisconnectReconnect()` crea una nuova CDP session dopo il detach, ma la vecchia session non viene esplicitamente chiusa. Se questa funzione viene chiamata ripetutamente, le session si accumulano.

```javascript
// Attuale — la vecchia session non viene chiusa
const session = await createCdpSession(page);
await session.detach();  // detach ma non close
const reconnected = await createCdpSession(page);  // nuova session, vecchia in memoria
```

### 2. Chrome version mismatch (playwrightEngine.js:7)
`DEFAULT_USER_AGENT` contiene "Chrome/141.0.0.0" ma il Chromium effettivamente lanciato potrebbe avere una versione diversa. I bot detector confrontano la versione nel UA con `navigator.userAgent` e `navigator.userAgentData`.

### Warning aggiuntivi
- Nessuna verifica che Page/Runtime domains siano effettivamente abilitati dopo reconnect
- Timing tra detach e reconnect è immediato — nessun delay, potenzialmente detectabile
- CDP lifecycle eseguito anche se non necessario — aggiunge latenza

### 1. Chiudere esplicitamente vecchia session
```javascript
async function applyCdpDisconnectReconnect(page) {
  const session = await createCdpSession(page);
  if (!session) return { used: false };
  
  try {
    await session.send("Page.enable").catch(() => undefined);
    await session.send("Runtime.enable").catch(() => undefined);
    
    if (typeof session.detach === "function") {
      await session.detach();
    }
  } finally {
    // Cleanup esplicita
    if (typeof session.dispose === "function") {
      await session.dispose().catch(() => undefined);
    }
  }
  
  // Delay realistico
  await new Promise(r => setTimeout(r, 50 + Math.random() * 100));
  
  const reconnected = await createCdpSession(page);
  // ...
}
```

### 2. Auto-detect Chrome version
```javascript
const browser = await ensureBrowser(config);
const version = await browser.version();
const chromeVersion = version.match(/Chrome\/(\d+)/)?.[1] || "131";
// Usare chromeVersion nel UA e nello stealth script
```

## Acceptance Criteria
- [ ] Vecchia CDP session chiusa esplicitamente prima di crearne una nuova
- [ ] UA version coerente con Chromium lanciato
- [ ] Delay realistico tra detach e reconnect (50-150ms)
- [ ] Test: verificare che dopo 10 cicli CDP, le session non si accumulano

## File da modificare
- `src/runtime/playwrightEngine.js:1115-1140` — CDP cleanup
- `src/runtime/playwrightEngine.js:7,1304` — UA version detection

## Dipendenze
Dopo #25 e #26.

**Comment by nikotheclawd** [comment]
Implemented and validated in `dev`.

### What changed
- `src/runtime/playwrightEngine.js`
  - **CDP lifecycle hardening** (`applyCdpDisconnectReconnect`):
    - added explicit session cleanup helper (`closeCdpSession`) with `detach` + `dispose`/`close` best-effort.
    - old session is now always cleaned before reconnect.
    - reconnected session is also explicitly detached/cleaned after domain enable.
    - added realistic reconnect delay window (**50-150ms**) via `computeCdpReconnectDelayMs`.
    - return payload now includes `pageEnabled/runtimeEnabled` flags for both phases.
  - **UA version consistency**:
    - added helpers `extractChromiumMajorVersion`, `applyChromiumMajorToUserAgent`, `resolveUserAgentFromBrowserVersion`.
    - when UA is not explicitly overridden, runtime now aligns UA `Chrome/<major>.0.0.0` with launched Chromium `browser.version()`.
    - explicit custom UA remains untouched.

### Tests
- `test/playwright-engine-cdp-lifecycle.test.js`
  - updated lifecycle checks to assert CDP sessions are cleaned and no active sessions leak.
  - added repeated-cycle regression (10 reconnect cycles, no session accumulation).
  - validates delay is actually applied.
- `test/playwright-engine-browser-engine.test.js`
  - added UA-version helper test.
  - added engine-level tests for default UA auto-alignment and custom-UA preservation.

### Validation
- `node --test test/playwright-engine-cdp-lifecycle.test.js test/playwright-engine-browser-engine.test.js` ✅ (16/16)
- `npm run test:fast` ✅ (180/180)
- `openclaw plugins doctor` ✅

### Commit
- `7c01dea` (`fix(cdp): close reconnect sessions and align UA major with chromium runtime (#28)`)
